drivers/dac/Kconfig.max22017

```
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Analog Devices Inc.
+# Copyright (c) 2024 BayLibre SAS
+# SPDX-License-Identifier: Apache-2.0
+
+config DAC_MAX22017
+	bool "Analog devices MAX22017 DAC"
```

Nits: Correct company name case (some more occurrences below)
```suggestion
	bool "Analog Devices MAX22017 DAC"
```